### DISCUSSION TITLE
Modelling of Plastic hardening (with necking)

### DISCUSSION THREAD

**Discussion by abarun22** [idaholab/moose#16199] [Q&A Modules: Solid mechanics] [answered]
Dear Wen,
Further to our earlier discussions at https://groups.google.com/g/moose-users/c/cWcA44gI9hQ/m/MgGaJPJjBAAJ
I would like to update that the new plasticity model built with PieceWise hardening function (PWH) gives reasonably asseptable results. Please see the attached slides. You could now see how realistic the plastic deformation is localizing heavily near the left symmetric face. The earlier problem with PWH has been bypassed by disallowing certain spurious states just after the yield. The results are looking good (with experiments and Ansys) at small strains, however showing a strange behaviour afterwards. There seems to be a stress drop, but with a linear variation, which looks pretty odd. Conversion to true Stress/Pl.Strain input hardening parameters at first place does not produce agreeable results. I would be extremely pleased if you could let me know thoughts on this behaviour.

PS. I am yet to perform a comparison with ANSYS at present due to some issues. Will let you know as soon as it is done.
[Moose_FEA_UG_16Nov2020.pptx](https://github.com/idaholab/moose/files/5546117/Moose_FEA_UG_16Nov2020.pptx)

Kind regards,
Arun

**Comment by GiudGiud** [comment, marked as answer]
@jiangwen84 @abarun22 can I close this? I imagine you guys had other occasions to talk about this over the last year

**Reply by abarun22** [reply]
Yes please, this can be considered closed now.
